test/e2e_federation/upgrade.go

```diff
@@ -25,7 +25,7 @@ import (
 	. "github.com/onsi/ginkgo"
 )
 
-var upgradeTests = []upgrades.Test{}
+var upgradeTests = upgrades.SimpleUpgradeTests()
```

Comment:
Since we will be adding upgrade tests for more than simple types, please add SimpleUpgradeTests to the upgradeTests slice instead assigning.